### PR TITLE
Fireproof ERT Hardsuits

### PR DESCRIPTION
**Pull request by Anderbergst**
As per request from this thread - http://hippiestation.com/showthread.php?tid=5572

Made ERT hardsuits fireproof.

### CODE REVIEW

**Comment by Swagile** [comment]
i know its a small change and all but did you test this

**Comment by Anderbergst** [comment]
I will test right now. 
Thanks for reminding!

**Comment by Anderbergst** [comment]
Works as intended. ERT will never burn to a fire anymore lol.

**Comment by Swagile** [comment]
GM + 1 then from me

File: html/changelogs/Anderbergst - FireproofERT.ylm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: Anderbergst
```
Comment (JohnGinnane) [inline html/changelogs/Anderbergst - FireproofERT.ylm]: Should be .yml

**Review by JohnGinnane — DISMISSED** [review]

**Comment by yobob591** [comment]
>ylm

kek

I like tho, ERT atm isnt very good unless they are robust which is rare

**Comment by KayeArray** [comment]
Fits nicely with my PR actually considering plasma sets people on fire.

**Comment by lcass** [comment]
should've been implemented in the first place , ERT overall needs a buffing.

**Comment by ArcLumin** [comment]
I like it. Code's good

**Comment by ghost** [comment]
Okay +1

**Comment by Swagile** [comment]
while your at it, can you also give them rad proof?

i can see traitors switching to dragging the SM behind them and radding ERT to death instead if they can't do easy plasma meme's anymore to kill ERT with

also prevents them from being KO's by singuloth (because 1 out of 5 rounds ERT is called, singuloth is loose)

**Comment by oprayx77** [comment]
didnt we stop sm dragging cause of erikhanson lol

**Comment by Swagile** [comment]
no idea if it was stopped, but this would also save ERT from getting singuloth'd as the rads coming out of it can stun you enough that you can't run away

**Comment by blobbernaut** [comment]
well deathsquad are there to maim kill burn, not fix an emergency

**Comment by Anderbergst** [comment]
I can make ERT hardsuits RAD PROOF too.

**Comment by ghost** [comment]
Cargo shards can be dragged still. SM core can't.

**Comment by KayeArray** [comment]
i miss dragging the SM *sniff

**Comment by Anderbergst** [comment]
I think ERT hardsuits are already rad proof since rad = 100

**Comment by SpankMasterson** [comment]
+1

**Comment by KayeArray** [comment]
just merge it already

**Comment by Swagile** [comment]
yeah but i believe armor is hardcoded not to be 100 even if you set it to 100

**Comment by ArcLumin** [comment]
Merging since HA

**Pull request merged**